src/video-player.js

```diff
@@ -390,7 +390,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
   _initAnalytics() {
     const analyticsOpts = this.playerOptions.analytics;
 
-    if (!window.ga && analyticsOpts) {
+    if (!window.ga && analyticsOpts && this.playerOptions.debug) {
```

Comment:
For later i think it is better to create an abstract that wraps the console.error statement and considers the debug mode.